src/networks.ts

```diff
@@ -280,6 +284,16 @@ export const PRODUCTION_NETWORKS: { [chainId: number]: PublicNetwork } = {
     oftEid: OFT_NO_EID,
     hypDomainId: MAINNET_CHAIN_IDs.MODE,
   },
+  [CHAIN_IDs.MONAD]: {
+    name: "Monad",
+    family: NONE, // Is this true?
```

Comment:
Yeah; `NONE` basically just means `EVM`. We have some refinement to do here, but for now it's OK.